test/gui/drone/comment.sh

```diff
@@ -5,35 +5,44 @@
 # $3 - DRONE_BUILD_NUMBER
 
 # create a github comment file only if guiTestUpload Folder is not empty
-if [[ $(find $1 -maxdepth 0 -empty) ]]; then
+if [[ $(find "$1" -maxdepth 0 -empty) ]]; then
     echo "No test results to upload"
     echo "Exiting..."
 
 else
     echo "creating comment for GUI log"
+
     # if there is index.html generated by squishrunner then create a comment indicating link of GUI result
-    if [[ -f $1/index.html ]]; then
-        echo ":boom: The GUI tests failed." >> $1/comments.file
-        echo "GUI Logs: (${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/index.html)" >> $1/comments.file
+    if [[ -f "$1/index.html" ]]; then
+        cat > "$1/comments.file" <<EOF
+
+:boom: The GUI tests failed.
+
+GUI Logs: ${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/index.html
+EOF
     fi
 
+    echo -e '\n<details>\n' >> "$1/comments.file"
+
     # if there is serverlog.log generated by squishserver then create a comment indicating link of server result
-    if [[ -f $1/serverlog.log ]]; then
+    if [[ -f "$1/serverlog.log" ]]; then
         echo "creating comment for server log"
-        echo "Server Logs: (${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/serverlog.log)" >> $1/comments.file
+        echo "Server Logs: ${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/serverlog.log" >> "$1/comments.file"
     fi
 
     # if there is stacktrace file then add to the comment
-    if [[ -f $1/stacktrace.log ]]; then
+    if [[ -f "$1/stacktrace.log" ]]; then
         echo "creating comment for stacktrace"
-        echo "Stacktrace: (${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/stacktrace.log)" >> $1/comments.file
+        echo "Stacktrace: ${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/stacktrace.log" >> "$1/comments.file"
     fi
 
-    if ! [[ $(find $1/screenshots -maxdepth 0 -empty) ]]; then
+    if ! [[ $(find "$1/screenshots" -maxdepth 0 -empty) ]]; then
         echo "creating comment for screenshots"
-        echo "Screenshots:" >> $1/comments.file 
-        for i in $(ls $1/screenshots); do
-            echo "- [$i](${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/screenshots/$i)" >> $1/comments.file
+        echo -e "\nScreenshots:" >> "$1/comments.file"
+        for i in "$1"/screenshots/*; do
+            echo "- [$i](${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/screenshots/$i)" >> "$1/comments.file"
```

Comment:
The screenshot link is for eg:  [/drone/src/test/guiReportUpload/screenshots/Adding_account_with_self_signed_certificate_for_the_first_time.png](https://cache.owncloud.com/public/owncloud/client/11404/guiReportUpload/screenshots//drone/src/test/guiReportUpload/screenshots/Adding_account_with_self_signed_certificate_for_the_first_time.png)

is invalid because the path `/drone/src/test/guiReportUpload/screenshots/` was appended to the link. Can you check it?

Nevertheless, I think it looks better if we put only the name of file instead of the whole path in the text for link to screenshots.

Reply:
Indeed, we have to call `basename` on `$i` in the square brackets.

Reply:
Can you run the full CI once? Just to verify the screenshot links are correct.